tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/manifest.ini

```diff
@@ -46,9 +46,8 @@ skip-if = device == "desktop" && os == "linux" && debug
 
 [test_gallery_switch_to_camera.py]
 # Bug 1111637 - Create a new test case for the switching to camera from gallery and back
-external = true
 camera = true
-sdcard = true
```

Comment:
Hrm, shouldn't test_gallery_switch_to_camera_main_view.py also had sdcard = true. Does the camera even start up when there is no storage?